src/qml/WelcomeScreen.qml

```diff
@@ -1,71 +1,311 @@
-import QtQuick 2.11
-import QtQuick.Controls 1.4
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Controls.Styles 1.2
+import QtQuick.Layouts 1.12
 import Theme 1.0
-import "."
 
-Item {
+Page {
+  property alias model: table.model
   signal showOpenProjectDialog
 
   Rectangle {
+    id: welcomeBackground
     anchors.fill: parent
-    color: "white"
+    gradient: Gradient  {
+      GradientStop  {
+        position: 0.0
+        color: "#ffdedede"
+      }
+      GradientStop  {
+        position: 0.33
+        color: "#00dedede"
+      }
+    }
+  }
 
-    Item {
+  ToolButton {
+    id: currentProjectButton
+    height: 56 * dp
+    width: 56 * dp
+    visible: false
+    anchors {
+      top: parent.top
+      left: parent.left
+    }
+    contentItem: Rectangle {
       anchors.fill: parent
-      anchors.margins: 10*dp
+      color: "transparent"
+      Image {
+        anchors.fill: parent
+        fillMode: Image.Pad
+        horizontalAlignment: Image.AlignHCenter
+        verticalAlignment: Image.AlignVCenter
+        source: Theme.getThemeIcon( 'ic_chevron_left_black_24dp' )
+      }
+    }
 
-      Row {
-        width: parent.width
+    onClicked: {
+      welcomeScreen.visible = false;
+      welcomeScreen.focus = false;
+    }
+  }
+  ScrollView {
+    padding: 0 * dp
+    ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+    ScrollBar.vertical.policy: ScrollBar.AsNeeded
+    contentItem: welcomeGrid
+    anchors.fill: parent
```

Comment:
```suggestion
    anchors.fill: parent
    horizontalScrollBarPolicy: Qt.ScrollBarAlwaysOff
```

Reply:
@m-kuhn , that's what's done on line 54 above, no?

Reply:
It flicks horizontally on my phone.
I think 54 is only for the visual scrollbar and not for the flicking functionality itself.

Reply:
![Screenshot from 2020-01-06 16-10-11](https://user-images.githubusercontent.com/1728657/71807924-092b7e00-309f-11ea-9a1c-7eaf608bbb6d.png)